fastlane/Fastfile

```diff
@@ -1,49 +1,105 @@
-# This file contains the fastlane.tools configuration
-# You can find the documentation at https://docs.fastlane.tools
-#
-# For a list of all available actions, check out
-#
-#     https://docs.fastlane.tools/actions
-#
-# For a list of all available plugins, check out
-#
-#     https://docs.fastlane.tools/plugins/available-plugins
-#
-
-# Uncomment the line if you want fastlane to automatically update itself
-# update_fastlane
+require_relative './amplify_pods.rb'
 
 opt_out_usage
-
 default_platform(:ios)
 
-COCOAPODS_PUSH_WAIT_TIME = 600
+PLIST_KEY = "CFBundleShortVersionString"
 
-pods = [
-  {spec: "Amplify.podspec", sleep: true},
-  # AmplifyPlugins depends on AWSPluginsCore, but skipping the sleep here
-  # since there's one after CoreMLPredictionsPlugin
-  {spec: "AWSPluginsCore.podspec", sleep: false},
-  {spec: "CoreMLPredictionsPlugin.podspec", sleep: true},
-  {spec: "AWSPredictionsPlugin.podspec", sleep: false},
-  {spec: "AmplifyPlugins.podspec", sleep: false}
-]
+COCOAPODS_PUSH_WAIT_TIME = 600
 
 platform :ios do
+  desc "Create a pre-release version by pushing a tag to GitHub, and pushing pods to CocoaPods"
+  lane :unstable do 
```

Comment:
I don't see an entry in `.circleci/config.yml` to trigger this lane. Is that coming in a future PR, or is it added in a way that doesn't reference the token, `"unstable"`?

Reply:
It's been added: https://github.com/aws-amplify/amplify-ios/pull/588/files#diff-1d37e48f9ceff6d8030570cd36286a61R163

I parameterized the deploy job. It passes unstable by default. For the full release, we're passing `release`, i.e., https://github.com/aws-amplify/amplify-ios/pull/588/files#diff-1d37e48f9ceff6d8030570cd36286a61R245

I can take out the default if that'll make it more straightforward

Reply:
No, parameters are a fine way to do this, I just didn't catch that